src/app/launcher/import-app/project-progress-importapp-nextstep/project-progress-importapp-nextstep.component.ts

```diff
@@ -26,7 +26,9 @@ export class ProjectProgressImportappNextstepComponent implements OnInit, OnDest
   }
 
   ngOnInit() {
-    this.subscriptions.push(this.projectProgressService.getProgress().subscribe((progress) => {
+    this.subscriptions.push(this.projectProgressService
+      .getProgress(this.launcherComponent.summary.uuidLink)
```

Comment:
Please do not store the uuidLink in the summary. Can we have the import progress component should fetch the uuidLink itself instead having the summary component store it?

Reply:
sure. i will work on it.